doc/cookbook/source/examples/classifier/multiclass_linearmachine.rst

```diff
@@ -0,0 +1,43 @@
+==========================
+Multi-class Linear Machine
+==========================
+
+We expand the application of linear machines to multi-class dataset by constructing generic multiclass classifiers with ensembles of binary classifiers.
+
+In this example, we show how to apply :sgclass:`CLibLinear` to multi-class cases with :sgclass:`CLinearMulticlassMachine`.
+
+See :cookbook_classifier:`linear_svm` for the infomration about :sgclass:`CLibLinear` binary classifier.
```

Comment:
I can only think of doing it case by case in conf.py..
like :cookbook_linear_svm: `*` redirects to a fixed page